### PR TITLE
complete ticket-6

### PR DESCRIPTION
**Pull request by megbuna**

### CODE REVIEW

File: src/components/CommentList.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        getComments(review_id).then((comments) => {
+            setListOfComments(comments)
+            setIsLoading(false)
+        });
+    }, []);
+
+    return isLoading ? (
+        <p> Loading comments... </p>
+    ) :
+    listOfComments.length === 0 ?  (<div><p> No comments yet </p>
```
Comment (lewispricey) [inline src/components/CommentList.jsx:21]: Really like the way you're checking the comments length and displaying a message if they don't exist 👍

File: src/components/CommentList.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    </div> )
+    :
+    (
+    <section className="commentList">
+        <h3> Comments: </h3>
+        {
+            listOfComments.map(({body, votes, author, created_at, comment_id, review_id}) => {
```
Comment (lewispricey) [inline src/components/CommentList.jsx:28]: Great use of destructuring here to grab just the information you need, keeps the code really clear!

File: src/components/CommentCard.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const CommentCard = ({body, votes, author, created_at}) => {
+    return (
+    <div className="commentCard">
```
Comment (lewispricey) [inline src/components/CommentCard.jsx:3]: Is there a more semantic HTML tag you could use here instead of a div?

**Review by lewispricey — CHANGES_REQUESTED** [review]
Great work so far Michael, checking the comments length and returning a message if there are no comments is a great way to indicate this to the user!

I'll send over a screenshot in a slack DM but when I tested your app on mobile the styling got very squewed and the text was very difficut to read. I'd recommend using mobile view on your browser and changing the styling with your view like this to fix it.

I haven't gone though the votes section as when I tested this the count was increasing on screen but not when I refreshed (this is the next ticket so not to worry)

Overall really good work, your code is very clear! Please fix the mobile view and resubmit a pull request once thats been done 😄

**Review by leekli — APPROVED** [review]
Thanks for making the styling changes, this looks better in mobile view - well done!

You are free to merge ticket 6.

**Pull request merged**